### ISSUE TITLE
Update example sizing notebooks with ConfigLoader

### ISSUE THREAD

**Issue by danielkberry** [mozilla/mozanalysis#251]
Now that metric references have been fully removed from Mozanalysis, some of the example experiment sizing Colab notebooks are out of date. We should update these.

**Comment by m-d-bowerman** [comment]
Agreed. We should incorporate Jared's updates, too, to demonstrate the (vastly improved) sizing outputs.
